packages/account/src/Components/poi/poi-country-selector/poi-country-selector.tsx

```diff
@@ -78,11 +91,31 @@ const CountrySelector = ({
                             'min-height': !is_from_external,
                         })}
                     >
-                        <Text className='proof-of-identity__header' align='center' weight='bold'>
-                            {localize('Proof of identity')}
-                        </Text>
+                        {failed_message ? (
+                            <React.Fragment>
+                                <Text className='' align='center' weight='bold' size={isMobile() ? 'xs' : 's'}>
+                                    <Localize i18n_default_text='Your identity verification failed because:' />
+                                </Text>
+                                <HintBox
+                                    icon='IcCloseCircleRed'
+                                    icon_height={16}
+                                    icon_width={16}
```

Comment:
Seems like the default icon size is 16, these 2 lines are unnecessary.

Reply:
Seems the changes got overridden while resolving conflicts